### PR TITLE
Add tooltip for installed with no videos

### PR DESCRIPTION
**Pull request by CodeyGuyDylan**
## Proposed changes:

Add tooltip for the following states of the videopress card
* Inactive with videos
* Active with no videos
* Active with a plan
* Active without a plan

These tooltips will have to be adjusted when we add the comparison data and when we add context switching between "last 30 days" and "all-time" stats. That will be handled in those individual PRs

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

P2: p1HpG7-rb7-p2

## Does this pull request change what data or activity we track or use?

Yes, there are tracks for all the tooltips onClick, as well as a track when the user clicks a link within one of the tooltips

## Testing instructions:

To see the tooltip copy you can look here: tCejO24bAGT2ME7iYEfmuG-fi-6392_3262

1. Create a Jurassic ninja site with sandbox access and checkout this branch on Jetpack (you may want to test this on your local environment, the stats are cached in a transient and it could be useful to comment that out when testing)
2. Go to My Jetpack. There should be no tooltips for VideoPress when the product is unowned and no videos are uploaded
image
3. Now add 1 or more videos to your site and go back to My Jetpack. You should now see a tooltip icon and when you open it, you should see the correct amount of videos in the content, as well as a link to some documentation
image
4. Make sure the `hosting` link goes to the right place (https://jetpack.com/support/jetpack-videopress/the-jetpack-videopress-dashboard/#manage-local-videos)
5. Now activate the product and connect your user account
6. To mock some randomized data, go to uggcf%3N%2S%2Suers.yv%2S%3Suggcf%3N%2S%2Sbcratebx.n8p.pbz%2Sfbhepr%2Skers%2Sjcpbz%2Schoyvp.ncv%2Serfg%2Sjcpbz%2Qwfba%2Qraqcbvagf%2Spynff.jcpbz%2Qwfba%2Qncv%2Qfgngf%2Qivqrb%2Qcynlf%2Qi1%2Q1%2Qraqcbvag.cuc%3Se%3Q210o65nn%26zb%3Q1199%26sv%3Q37%2350-og and replace all those zeros with something like rand(0, 100). You can pick whatever numbers you'd like if you wanted to test out smaller or bigger numbers.
7. Make sure you only have 1 video and look at the tooltips. They should both be in the context of a site without a plan and only 1 video
image
image
8. Now purchase a VideoPress plan and go back to My Jetpack. The tooltip related to views should now be updated.
image

Bonus: Test with an active plugin with 0 stats and 1 video. Ensure that we do not imply they are currently gathering views on their video
image

### CODE REVIEW

File: projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/use-videopress-description.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ const useVideoPressCardDescription = ( {
 	if ( isPluginActive && ! videoCount ) {
 		return preventWidows(
 			__(
-				'Stunning-quality, ad-free video in the WordPress Editor. Begin by uploading your first video.',
+				'Stunning-quality, ad-free video in the WordPress Editor. Begin by uploading your first video',
```
Comment (CodeyGuyDylan) [inline projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/use-videopress-description.ts]: The info icon looked weird after a period, let me know if you disagree

File: projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/use-tooltip-copy.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+		} );
+	}, [ recordEvent, hostingRedirectLink ] );
+
+	const inactiveWithVideos = {
+		title: __( 'The finest video for WordPress', 'jetpack-my-jetpack' ),
+		text: createInterpolateElement(
+			sprintf(
+				// translators: %d is the number of videos in the Media Library that could benefit from VideoPress.
+				_n(
+					'You have %d video in your Media Library that could benefit from VideoPress. Start <a>hosting</a> them today to unlock multiple benefits: enhanced quality add-free streaming, faster load times, customizable player controls.',
```
Comment (jboland88) [inline projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/use-tooltip-copy.ts]: In this string: "them" sounds incorrect for the singular.

File: projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/use-tooltip-copy.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+		title: __( 'The finest video for WordPress', 'jetpack-my-jetpack' ),
+		text: __(
+			'Give your videos a boost! 🚀 Try hosting with VideoPress for superior quality and performance.',
+			'jetpack-my-jetpack'
+		),
+	};
+
+	const viewsWithoutPlan = {
+		title: __( 'High-quality video, wherever your audience is', 'jetpack-my-jetpack' ),
+		text: __( 'Success! 🌟 Your video is live and gathering views.', 'jetpack-my-jetpack' ),
```
Comment (jboland88) [inline projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/use-tooltip-copy.ts:56]: I think this bug was covered in the other PR, but I am still getting this message with no watch time or views. I think the intent is to omit the stats with 0 views?

**Review by jboland88 — DISMISSED** [review]
This looks pretty good and tested well.
The way you are managing the tooltip copy makes sense to me.

I noticed one grammar error and one bug that I think will be addressed in the base branch. This one seems good to merge into base IMO.

**Comment by CodeyGuyDylan** [comment]
@ilonagl this is the PR where the tooltip comes into play. Let me know if the placements look okay!

**Comment by ilonagl** [comment]
Thank you for the ping, @CodeyGuyDylan! 

> 
image

Could we make this text shorter so it fits into two lines? Something like this: "This metric shows total video viewing time for the last 30 days."

The info icon placement looks good. It would be best if we could keep them consistent. The rest of them have info icon next to the header, so it would be good to make it like this in this case as well: 

image


What about making the header 'Existing videos' and placing the info icon next to it?

Other than that, looks great. Thank you for working on this!!

**Comment by ilonagl** [comment]
@elliottprogrammer could we possibly update the info icon placement for the Protect card?

File: projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/use-tooltip-copy.ts (hunk 2 above)
Comment (CodeyGuyDylan) [reply projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/use-tooltip-copy.ts]: Nice catch, updating it to "it"

File: projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/use-tooltip-copy.ts (hunk 3 above)
Comment (CodeyGuyDylan) [reply projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/use-tooltip-copy.ts:56]: Talked about this elsewhere, decided to keep this way 😄

File: projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/use-videopress-description.ts (hunk 1 above)
Comment (elliottprogrammer) [reply projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/use-videopress-description.ts]: Hmm, yeah IMO I think the period should remain. To me it's the opposite, and to me it looks kinda weird without the period. (and seems grammatically improper)

I'd be curious what anybody else thinks. 🤷‍♂️

With Period | Without Period
--- | ---
Screen Shot on 2024-08-16 at 11-57-00 | Screen Shot on 2024-08-16 at 11-57-57

Not a blocker though... 👍

File: projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/videopress-value-section.tsx [hunk 4]
```diff
@@ -46,8 +108,26 @@ const VideoPressValueSection: FC< VideoPressValueSectionProps > = ( { isPluginAc
...
+					<InfoTooltip
+						tracksEventName="videopress_card_tooltip_open"
+						tracksEventProps={ {
+							location: 'watch_time',
+							current_watch_time: currentWatchTime,
+							...tracksProps,
+						} }
+					>
+						<h3>{ watchTime.title }</h3>
+
```
Comment (elliottprogrammer) [inline projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/videopress-value-section.tsx]: Nitpick: Unnecessary space here...
Not a blocker, of course. 😉

**Review by elliottprogrammer — DISMISSED** [review]
Tested functionality works good as described! The code looks overall good to me, I don't see any issues! Just a couple minor comments.
LGTM! 👍

**Review by elliottprogrammer — APPROVED** [review]

**Pull request merged**